tombstones/tombstones.go

```diff
@@ -30,7 +32,7 @@ import (
 	"github.com/prometheus/tsdb/fileutil"
 )
 
-const tombstoneFilename = "tombstones"
+const TombstoneFilename = "tombstones"
```

Comment:
again, with `tombstones` package name you don't need `tombstones` prefix everywhere, we can simplify (: